template/webpack.config.prod.js

```diff
@@ -24,34 +20,32 @@ module.exports = {
   ],
 
   output: {
-    path: 'public/dist/',
-    pathInfo: true,
+    path: path.join(__dirname, '/public/dist/'),
+    pathinfo: true,
     publicPath: '/dist/',
     filename: 'bundle.[hash].min.js',
   },
 
   resolve: {
-    root: path.join(__dirname, ''),
-    modulesDirectories: [
-      'web_modules',
-      'node_modules',
-      'assets',
-      'assets/components',
-    ],
-    extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx'],
+    modules: [__dirname, 'node_modules'],
+        alias:{
```

Comment:
(pedantic) extract tab here before `alias`

Reply:
not sure I understand?